config/gplus-fl8005a.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From stock recovery GPLUS_FL8005A_V0.6.9.0
+
+OPTIONS=(-r power --dumb-dcs)
+PANELS=(
+	[hx8394d_800p_video]="gplus,hx8394d"
```

Comment:
This feels a bit too generic, maybe

```suggestion
	[hx8394d_800p_video]="gplus,fl8005a-hx8394d"
```